packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -745,109 +748,134 @@ class GraphiQLWithContext extends React.Component<
-                          );
-                        }}>
-                        Query Variables
-                      </div>
-                      {headerEditorEnabled && (
-                        <div
-                          style={{
-                            marginLeft: '20px',
-                          }}
-                          className={`variable-editor-title-text${
-                            this.state.activeSecondaryEditor === 'header'
+                        }}
+                        onCopyQuery={this.props.onCopyQuery}
+                        onEdit={this.props.onEditQuery}
+                        onEditOperationName={this.props.onEditOperationName}
+                        readOnly={this.props.readOnly}
+                        validationRules={this.props.validationRules}
+                      />
+                    </section>
+                  </div>
+                  <div ref={this.props.editorToolsResize.dragBarRef}>
+                    <div className="graphiql-editor-tools">
+                      <div className="graphiql-editor-tools-tabs">
+                        <UnStyledButton
+                          className={`graphiql-editor-tools-tab-button ${
+                            this.state.activeSecondaryEditor === 'variable'
                               ? ' active'
                               : ''
                           }`}
                           onClick={() => {
                             if (
-                              this.props.secondaryEditorResize.hiddenElement ===
+                              this.props.editorToolsResize.hiddenElement ===
                               'second'
                             ) {
-                              this.props.secondaryEditorResize.setHiddenElement(
+                              this.props.editorToolsResize.setHiddenElement(
                                 null,
                               );
                             }
                             this.setState(
                               {
-                                activeSecondaryEditor: 'header',
+                                activeSecondaryEditor: 'variable',
                               },
                               () => {
-                                this.props.editorContext.headerEditor?.refresh();
+                                this.props.editorContext.variableEditor?.refresh();
                               },
                             );
                           }}>
-                          Request Headers
-                        </div>
-                      )}
+                          Variables
+                        </UnStyledButton>
+                        {this.props.headerEditorEnabled ? (
+                          <UnStyledButton
+                            className={
+                              this.state.activeSecondaryEditor === 'header'
+                                ? ' active'
```

Comment:
```suggestion
                                ? 'active'
```

Reply:
Good catch, same for the button for variables (the other class name there was unused) 👍